src/app/models/querySchools.js

```diff
@@ -0,0 +1,73 @@
+import settings from 'settings';
+
+const MAX_SEARCH_RESULTS = 1000;
+
+export function generateCityState(schoolInfo) {
+    return [schoolInfo.physical_city, schoolInfo.physical_state_province].filter((a) => a).join(', ');
+}
+
+function augmentInfo(item) {
+    const result = {
+        pk: item.pk,
+        cityState: generateCityState(item.fields),
+        institutionalPartner: item.fields.key_institutional_partner,
+        institutionType: item.fields.type,
+        fields: item.fields
+    };
+    const lngLat = [Number(item.fields.long), Number(item.fields.lat)];
+
+    if (!(lngLat[0] === 0 && lngLat[1] === 0)) {
+        result.lngLat = lngLat;
+    }
+    if (item.fields.testimonial) {
+        result.testimonial = {
+            text: item.fields.testimonial,
+            name: item.fields.testimonial_name,
+            position: item.fields.testimonial_position
+        };
+    }
+
+    return result;
+}
+
+const searchPath = `${settings.apiOrigin}${settings.apiPrefix}/schools/`;
+const toSearchParam = {
+    partners: 'key_institutional_partner',
+    savings: 'saved_one_million',
+    testimonials: 'testimonial',
+    'institution-type': 'type'
+};
+
+function doFetch(searchParameters) {
+    return fetch(`${searchPath}?${searchParameters}`)
+        .then((response) => response.json())
+        .then((response) =>
+            response.length > MAX_SEARCH_RESULTS ? [] :
+                response.map(augmentInfo)
+                    .sort((a, b) => a.fields.name.toLowerCase() < b.fields.name.toLowerCase() ? -1 : 1)
+        );
+}
+
+export default function (queryString, selectedFilters) {
```

Comment:
This function can also use documentation (like indicating that is is communicating with CMS for API requests).